src/site/content/en/blog/aurora-resource-inlining/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: 'Resource Inlining in JavaScript frameworks'
+subhead: Improving Largest Contentful Paint across the JavaScript ecosystem.
+authors:
+  - mgechev
+date: 2021-06-28
+hero: image/S838B7UEsdXmwrD8q5gvNlWTHHP2/yXASsFeUg39y0K7aFJIY.jpg
+alt: A labyrinth.
+description: Learn about the latest optimizations implemented in JavaScript frameworks in collaboration with project Aurora.
+tags:
+  - blog
+  - angular
+  - next.js
+  - nuxt.js
+  - cwv
+  - lcp
```

Comment:
```suggestion
  - web-vitals
```

Reply:
@mgechev The tags above are not supported, we try to keep a lean tag list. If you have a strong reason for adding a new tag, you can open an issue and submit a PR.

@rachelandrew Check for tag validity, here's where you can find the supported tags: https://web.dev/handbook/yaml-front-matter/#supported-keywords